### PR TITLE
This PR adds tests for  3scale-connect-api(Staging)

### PR DESCRIPTION
**Pull request by GeetikaBatra**

### CODE REVIEW

File: integration-tests/feature_list.txt [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 #features/anitya.feature
 #features/stack_analyses_v2.feature
-features/selfcheck.feature
+#features/selfcheck.feature
 #features/regression_tests.feature
-features/user_tag.feature
-features/gremlin.feature
+#features/user_tag.feature
+#features/gremlin.feature
+features/three_scale.feature
```
Comment (samuzzal-choudhury) [inline integration-tests/feature_list.txt]: @GeetikaBatra why are all existing features commented out? All features must run during integration test along with the one that you've added.

File: integration-tests/cico_run_tests.sh [hunk 2]
```diff
@@ -11,6 +11,8 @@ docker run -t \
     -e F8A_JOB_API_URL=${F8A_JOB_API_URL} \
     -e F8A_ANITYA_API_URL=${F8A_ANITYA_API_URL} \
     -e F8A_GREMLIN_URL=${F8A_GREMLIN_URL} \
+    -e F8A_3SCALE_URL=${F8A_3SCALE_URL}
```
Comment (samuzzal-choudhury) [inline integration-tests/cico_run_tests.sh]: @GeetikaBatra a backslash is missing here

Comment (GeetikaBatra) [reply integration-tests/cico_run_tests.sh]: @samuzzal-choudhury, thanks for pointing out.

**Comment by GeetikaBatra** [comment]
[test]

**Comment by GeetikaBatra** [comment]
Please let me know the modifications, I will update the code.
Thanks for merging.

**Comment by tisnik** [comment]
@GeetikaBatra thanks for this addition + FYI: I've made some minor updates in your tests: https://github.com/fabric8-analytics/fabric8-analytics-common/pull/410

**Comment by GeetikaBatra** [comment]
@tisnik , your welcome and thanks for the updates. https://github.com/fabric8-analytics/fabric8-analytics-common/pull/410, LGTM +1

**Pull request merged**